.github/workflows/reviewers.yml

```diff
@@ -0,0 +1,39 @@
+reviewers:
+  # The default reviewers
+  defaults:
+    - LysandreJik
+    - osanseviero
+  # Reviewers per author.
+  # Keys are reviewees, each of which has an array of reviewers.
+  per_author:
+    core-library:
+      - muellerzr
+    documentation:
+      - stevhliu
+    community-inference:
+      - Narsil
+    community-tasks:
+      - merveenoyan
+    widgets:
+      - mishig25
```

Comment:
My understanding is that this specifies that every time someone from `key` opens a PR, the values will review. For example, for people from the `core-library` group, `muellerzr` would review. But I don't think there is `core-library` group and afaik we don't need to auto-assign reviewers based on author  

https://github.com/marketplace/actions/auto-request-review#auto-assign-reviewers-based-on-the-author

Reply:
So if I understand correctly `per_author` should likely be `groups` instead, no?

Reply:
Correct! Thanks for missing that. Will fix nit